### PR TITLE
Deprecate v2alpha1 and v2alpha2

### PR DESCRIPTION
**Pull request by renzodavid9**
Deprecating v2alpha1 and v2alpha2 formatter version.

Related integration tests were removed. Documentation updated.

# Submitter Checklist

As the author of this PR, please check off the items in this checklist:

- [x] Has [Docs](https://github.com/tektoncd/community/blob/main/standards.md#docs) included if any changes are user facing
- [x] Has [Tests](https://github.com/tektoncd/community/blob/main/standards.md#tests) included if any functionality added or changed
- [x] Follows the [commit message standard](https://github.com/tektoncd/community/blob/main/standards.md#commits)
- [ ] Meets the [Tekton contributor standards](https://github.com/tektoncd/community/blob/main/standards.md) (including
  functionality, content, code)
- [ ] Release notes block below has been updated with any user facing changes (API changes, bug fixes, changes requiring upgrade notices or deprecation warnings)
- [ ] Release notes contains the string "action required" if the change requires additional action from users switching to the new release

# Release Notes

``` release-note
Deprecation of v2alpha1and v2alpha2 provenance formats. Use v2alpha3, v2alpha4 instead.
```

### CODE REVIEW

File: pkg/config/config.go [hunk 1]
```diff
@@ -266,12 +266,12 @@ func NewConfigFromMap(data map[string]string) (*Config, error) {
 	if err := cm.Parse(data,
 		// Artifact-specific configs
 		// TaskRuns
-		asString(taskrunFormatKey, &cfg.Artifacts.TaskRuns.Format, "in-toto", "slsa/v1", "slsa/v2alpha1", "slsa/v2alpha2", "slsa/v2alpha3"),
+		asString(taskrunFormatKey, &cfg.Artifacts.TaskRuns.Format, "in-toto", "slsa/v1", "slsa/v2alpha1", "slsa/v2alpha3"),
```
Comment (chitrangpatel) [inline pkg/config/config.go:269]: remove v2alpha1 as well.

Comment (renzodavid9) [reply pkg/config/config.go]: Done.

**Review by lcarva — APPROVED** [review]
/lgtm

**Pull request merged**